lib/board.rb

```diff
@@ -0,0 +1,59 @@
+class Board
+  attr_accessor :cells
+  attr_reader :corners
```

Comment:
@Juanriv45  is corners needed or are we just adding more complexity than we really need?